qtpy/QtTextToSpeech.py

```diff
@@ -7,11 +7,13 @@
 
 """Provides QtTextToSpeech classes and functions."""
 
+from packaging.version import parse
 from . import (
```

Comment:
```suggestion
from qtpy import (
```

All the other test modules I could find use absolute imports, so may as well be consistent (as well as with other files in this PR)